pyramid_oereb/contrib/print_proxy/mapfish_print.py

```diff
@@ -142,6 +142,34 @@ def __call__(self, value, system):
             restriction_on_landownership['Reference'] = reference
             restriction_on_landownership['Article'] = article
 
+        # One restriction entry per theme
+        theme_restriction = {}
+        text_element = [
+            'Information', 'Lawstatus_Code', 'Lawstatus_Text', 'ResponsibleOffice_Name',
+            'ResponsibleOffice_OfficeAtWeb', 'SymbolRef', 'TypeCode'
+        ]
+        for restriction_on_landownership in extract_dict.get('RealEstate_RestrictionOnLandownership', []):
+            theme = restriction_on_landownership['Theme_Code']
+            if theme not in theme_restriction:
+                theme_restriction[theme] = restriction_on_landownership
+                for element in text_element:
+                    restriction_on_landownership[element] = set([restriction_on_landownership[element]])
+                continue
+            curent = theme_restriction[theme]
+            # Number or array
+            for element in ['Area', 'Article', 'LegalProvisions', 'PartInPercent', 'Reference']:
+                curent[element] += restriction_on_landownership[element]
+            # Text
+            for element in text_element:
+                curent[element] += restriction_on_landownership[element]
```

Comment:
My instance says: TypeError: unsupported operand type(s) for +=: 'set' and 'unicode'

Reply:
Should be fixed :-)